src/ImageDeleteModal.jsx

```diff
@@ -81,11 +74,20 @@ export const ImageDeleteModal = ({ imageWillDelete, onAddNotification }) => {
         }
     };
 
+    const imageName = repoTags?.[0].split(":")[0].split("/").at(-1) ?? "";
```

Comment:
This definitely makes more assumptions than before, which was just `repoTags ? repoTags[0] : ""` before. So it's more likely to break.